src/migrations/20200305133048-subscription.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('subscription', {
```

Comment:
Let's call this "groupSubscription". The word Subscription makes me think on a GraphQL Subscription (which is something we may use in a future).